projects/angular/src/forms/datepicker/providers/date-navigation.service.ts

```diff
@@ -68,9 +70,11 @@ export class DateNavigationService {
    * Notifies that the selected day has changed so that the date can be emitted to the user.
    * Note: Only to be called from day.ts
    */
-  notifySelectedDayChanged(dayModel: DayModel) {
+  notifySelectedDayChanged(dayModel: DayModel, emitEvent = true) {
```

Comment:
We can pass an object instead of just a boolean so that this is more readable at the call site. See e88d1dd8ceacd155ed5df61f984559105cba2075 for an example.